srcpkgs/gocryptfs/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'gocryptfs'
+pkgname=gocryptfs
+version=1.6
+revision=1
+wrksrc="${pkgname}_v${version}_src-deps"
+build_style=go
+go_import_path="github.com/rfjakob/gocryptfs"
+hostmakedepends="pandoc man-db"
+depends="fuse"
+short_desc="Encrypted overlay filesystem written in Go"
+maintainer="mustaqim <mustaqim+void@pm.me>"
+license="MIT"
+changelog="https://github.com/rfjakob/gocryptfs#changelog"
+homepage="https://nuetzlich.net/gocryptfs"
+distfiles="https://github.com/rfjakob/${pkgname}/releases/download/v${version}/${pkgname}_v${version}_src-deps.tar.gz"
+checksum=98ff3bc0497ca195f65fa5912ebab33a0f80aa27205b0b35e73908d23dcefbe6
+
+do_build() {
+	$GOSRCPATH/build-without-openssl.bash
```

Comment:
Have you considered using `go_build_tags=without_openssl` and removing this function? I say this because overriding `do_build()` is likely leading to the cross compilation errors.

Reply:
yes, most build styles try to provide the necessary variables, arguments, and what not for cross compilation to work without extra effort.